src/prognostic_equations/vertical_diffusion_boundary_layer.jl

```diff
@@ -17,37 +17,89 @@ import ClimaCore.Operators as Operators
 # 1) turn the liquid_theta into theta version
 # 2) have a total energy version (primary goal)
 
-function eddy_diffusivity_coefficient(C_E::FT, norm_v_a, z_a, p) where {FT}
-    p_pbl = FT(85000)
-    p_strato = FT(10000)
-    K_E = C_E * norm_v_a * z_a
-    return p > p_pbl ? K_E : K_E * exp(-((p_pbl - p) / p_strato)^2)
+function compute_boundary_layer_height(Ri_c::FT) where {FT}
+    # Needs a column search given colidx
+    return nothing
+end
+function compute_bulk_richardson_number(
+    θ_v::FT,
```

Comment:
Is `θ_v` θ_v(z) and `θ_a` θ_v(z_a). Since `v` refers to virtual for both, I wonder if we could use `θ_z` or something for the first one, or we could add comments next to the arguments?